src/main.rs

```diff
@@ -82,21 +81,23 @@ fn setup(
     commands
         .spawn((Collider::cuboid(cube_size * 0.5, cube_size * 0.5, cube_size * 0.5), RigidBody::Dynamic))
         .insert(ColliderMassProperties::Mass(1.0))
+        .insert(Damping { linear_damping: 0.1, angular_damping: 0.1 })
```

Comment:
Remove this, boxes of different masses should fall at the same acceleration and damping is (probably) not necessary for this project.